### ISSUE TITLE
find games by game shape?

### ISSUE THREAD

**Issue by projectcybersyn1** [NotJoeMartinez/chess.com-brilliant-move-finder#1]
I wonder if it'd be possible to find games by "game shape" designations. Cool project by the way! I hope this is an appropriate place to leave a comment like this, I'm not very familiar with github.

**Comment by NotJoeMartinez** [comment]
TLDR: yes, but it will be a bit more complicated than finding the brilliant games.

This script works because it use the API endpoint that chess.com uses to create this graph has a node called `movesByClassificationOverTime`. `https://www.chess.com/service/insights/{uuid}/{TARGET_TIME_CLASS}/all-time?uuid={uuid}`

<img width="715" alt="image" src="https://github.com/user-attachments/assets/72547b94-72fd-48bf-bec6-56a0172c9261">


```json
    "movesByClassificationOverTime": {
        "2022-02-10": {
            "best": 101,
            "excellent": 93,
            "good": 39,
            "book": 21,
            "inaccuracy": 32,
            "mistake": 25,
            "blunder": 28,
            "missedwin": 15,
            "forced": 2
        },
```
It doesn't have a node like this for game shape, it just returns the totals under the `gameArch` node 
```json

"gameArc": {
        "balancedWinDrawLoss": {
            "win": 11,
            "draw": 0,
            "loss": 1
        },
        "smoothWinDrawLoss": {
            "win": 5,
            "draw": 0,
            "loss": 7
        },
etc..
```
think out loud here...

It still looks like you can control the date range of these totals by using the `from` and `to` query prams. These query
prams are not dates though, they are integers where `to=1` represents the most recent date where a game was played and 
`from={integer}` represents the sequential positive index of the "game date" backwards of when a game was played. In 
the past three years I've played about 2000 rapid games and those games were played on about 1000 individual dates. 
It's possible but a script would need to fetch those individual days from the API and match them up to their 
appropriate from/to index to accomplish this, I'll ping you if I get around it.
